lib/puppet_x/vmware/mapper/virtual_disk_map.rb

```diff
@@ -0,0 +1,65 @@
+# Copyright 2015 VMware, Inc.
+
+require 'set'
+
+module PuppetX::VMware::Mapper
+
+  class VirtualDiskMap < Map
+    def initialize
+      @initTree = {
+        Node => NodeData[
+          :node_type => 'VirtualDisk',
+        ],
+        :key           => LeafData[
+          :desc          => 'Only used internally when a update or removal to an existing monitor',
+        ],
+        :controllerKey => LeafData[
+          :desc          => 'Object key for the controller object for this device. This property contains the key property value of the controller device object.',
+          :prop_name     => 'controller'
+        ],
+        :capacityInKB => LeafData[
```

Comment:
Why use deprecated capacityInKB instead of capacityInBytes (or at least allow either)?

http://pubs.vmware.com/vsphere-55/topic/com.vmware.wssdk.apiref.doc/vim.vm.device.VirtualDisk.html